scheme.go

```diff
@@ -65,6 +65,11 @@ func schemeFromFile(schemesFS fs.FS, fileName string) (*scheme, bool) {
 		ok = false
 	}
 
+	// Sanitize any fields which were added later
+	if ret.Description == "" {
```

Comment:
Does this cover empty as well as excluded entirely?

Reply:
Yes - it covers both. Go will default Description to the "zero value" if it's not there, which is an empty string.